### DISCUSSION TITLE
jwt解析器文本框太小，不能一次展示所有信息，需要拖动滚动条才行，不太方便

### DISCUSSION THREAD

**Discussion by jackehuang** [he3-app/he3-feedback#87] [Bug Report] [answered]
![image](https://user-images.githubusercontent.com/17893904/227826360-785ca2b2-ccec-4173-aafb-45afa6ef9511.png)

**Comment by zhenweiwang1990** [comment, marked as answer]
这个问题预计下个版本就会有改善了，多谢建议。
